src/components/MasterChart/MasterChart.js

```diff
@@ -8,6 +8,9 @@ import _ from 'lodash';
 import DataParser from '../../utils/DataParser';
 import GraphTutorialPopover from '../GraphTutorialPopover/GraphTutorialPopover';
 import './MasterChart.css';
+import {Panel} from 'react-bootstrap';
```

Comment:
absolute imports should be before relative imports

Reply:
Problem resolved, can you please check?